File: Modul-3/Soal-4/PRAK304-2310817110013-RANDY_FEBRIAN.py
for i in range(5):
    nilai = int(input())
    if(nilai < 0):
        break
    elif(nilai == 0):
        print("Nol")
    elif(0 < nilai < 10):
        print("Satuan")
    elif(10 < nilai < 20):
        print("Belasan")
    elif(20 <= nilai < 100 or nilai == 10):
        print("Puluhan")
    elif(nilai <= 100):
        print("Anda Menginput Melebihi Limit Bilangan")

File: Modul-3/Soal-4/PRAK304-2310817110013-RANDY_FEBRIAN.c
#include <stdio.h>

int main(){
    int a;

    for(int i = 0; i < 5; i++){
        scanf("%d", &a);
        if (a < 0){
            break;
        } else if (a == 0){
            printf("Nol\n");
        } else if(a > 0 && a < 10){
            printf("Satuan\n");
        } else if (a > 10 && a < 20){
            printf("Belasan\n");
        } else if (a >= 20 && a < 100 || a == 10){
            printf("Puluhan\n");
        } else{
            printf("Anda menginput Melebihi Limit Bilangan\n");
        }
    }
    return 0;
}